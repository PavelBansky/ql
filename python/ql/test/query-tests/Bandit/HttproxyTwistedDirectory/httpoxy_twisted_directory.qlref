Bandit/httpoxy_twisted_directory.ql