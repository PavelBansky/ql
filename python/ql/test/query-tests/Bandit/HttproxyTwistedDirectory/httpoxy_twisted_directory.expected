| Test.py:5:26:5:43 | Attribute | Consider possible security implications associated with twisted.web.twcgi.CGIDirectory module. |
